### DISCUSSION TITLE
Assign Static IP to Flowise in Azure

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#2480] [Q&A]
Hi team,
I am trying to depoy flowise in Azure but time by time the IP is modified. How can I make to have a fix IP address deploying it? Thanks in advance.

**Comment by amansoni7477030** [comment]
Hii @claclin 
To configure the IP address in your Cloud-Hosted Flowise Service, start by creating an environment variable named `NUMBER_OF_PROXIES` and set its value to `0` in your hosting environment. Restart the Flowise service to apply these changes. Next, verify the IP address by accessing **{{hosted_url}}/api/v1/ip** either through your web browser or via an API request. Compare the returned IP address with your current IP address, which you can find at [ip.nfriedly.com](http://ip.nfriedly.com/) or [api.ipify.org](https://api.ipify.org/).

If the IP addresses do not match, increment the `NUMBER_OF_PROXIES` variable by `1` and restart the Flowise service. Repeat this process until the returned IP address matches your current one. If this method does not work, you may need to modify the settings in your **NGINX** configuration file to achieve the correct IP address setup.
